samples/rag-cosmosdb/python/local.settings.json

```diff
@@ -5,9 +5,10 @@
     "AzureWebJobsFeatureFlags": "EnableWorkerIndexing",
     "FUNCTIONS_WORKER_RUNTIME": "python",
     "CosmosDBMongoVCoreConnectionString": "",
-    "AZURE_OPENAI_ENDPOINT": "https://<resource-name>.openai.azure.com/",
-    "CHAT_MODEL_DEPLOYMENT_NAME": "gpt-3.5-turbo",
-    "EMBEDDING_MODEL_DEPLOYMENT_NAME": "text-embedding-ada-002",
-    "PYTHON_ISOLATE_WORKER_DEPENDENCIES": "1"
+    "AZURE_OPENAI_ENDPOINT": "",
+    "CHAT_MODEL_DEPLOYMENT_NAME": "gpt-35-turbo",
+    "EMBEDDING_MODEL_DEPLOYMENT_NAME": "vector_search_task",
+    "PYTHON_ISOLATE_WORKER_DEPENDENCIES": "1",
+    "AZURE_OPENAI_KEY": ""
```

Comment:
don't modify any default values in the local.settings.json of samples unless you are introducing any new setting